src/utils/advancedSearchUtils.ts

```diff
@@ -74,3 +62,44 @@ export const buildGoBackHref = (referer) => {
   if (!goBackEndpoint) return "/"
   return goBackEndpoint
 }
+
+export function mapCollectionsIntoLocations(collections) {
+  return searchVocabularies.buildingLocations
+    .filter((building) => building.value !== "rc")
+    .map((building) => {
+      const children = collections
+        .filter((col) => col.value.startsWith(building.value))
+        .map((col) => ({
+          id: col.value,
+          name: col.count
+            ? `${col.label} (${col.count.toLocaleString()})`
+            : col.label,
+          ...(col.count !== undefined ? { count: col.count } : {}),
+        }))
+      return {
+        id: building.value,
+        name: building.label,
+        children,
+      }
+    })
+    .filter((group) => group.children.length > 0)
+}
+
+export function mapCollectionToFilterTag(collectionValue, collectionName) {
```

Comment:
Can you add a comment explaining what this method does? And/or move to the applied filter utils file, since it appears to only be used there.